pom.xml

```diff
@@ -66,7 +66,21 @@
         <scala.binary.version>2.12</scala.binary.version>
         <maven.version>3.6.3</maven.version>
         <spark.version>3.0.1</spark.version>
-        <log4j.version>1.7.30</log4j.version>
+        <slf4j.version>1.7.30</slf4j.version>
+        <mockito.verison>1.10.19</mockito.verison>
+
+        <maven.plugin.antrun.version>3.0.0</maven.plugin.antrun.version>
```

Comment:
The above version change looks fine. But I am -1 with these plugin changes. Not that straightforward as we only use them once each

Reply:
In my view, there are 2 benefits:
1. unify both dependencies and plugins versions to one place
2. if we want to try another version of plugin, just build with `-Dmaven.plugin.xxx.version=1.2.3`

Reply:
thanks for clarification